### DISCUSSION TITLE
Change App ID for Action Builds

### DISCUSSION THREAD

**Discussion by AkiraKusanagi** [EmiyaSyahriel/CrossLauncher#61] [General]
Just a thought. So you could test some of the features in the action build without having to delete the release build installed on the device.

**Comment by EmiyaSyahriel** [comment]
I'll investigate this if this is possible, my current prediction is that I have to add another variant to the build config that have different package name (or prefix at least) and call the corresponding `assemble[Variant]` instead of `assembleDebug` when building from action build as asked [here in StackOverflow](https://stackoverflow.com/questions/75957054/android-different-applicationid-applicationidsuffix-by-flavors-chains).

Anyway, I'll make the issue entry for this discussion.

**Reply by AkiraKusanagi** [reply]
Alright, thanks!

**Reply by EmiyaSyahriel** [reply]
Implemented, can you try it? It's in the [`Add "DEX" variant for Action Build (#68)`](https://github.com/EmiyaSyahriel/CrossLauncher/actions/runs/6991828586) action.

Please keep in mind that since the package name is changed, the app's data folder is also separated from the release page version one and you might need to migrate the customization files in-between. The issue page is #63 btw.
